.github/scripts/Rename_Photos

```diff
@@ -0,0 +1,26 @@
+#This script renames photos from BHVP_PH_CetaitParis_DP_1538_01_015 and BHVP_PH_CetaitParis_DP_1538_01_016.jpg to 16_slide.jpg and 16_photo.jpg
+
+import math
+import os
+import csv
+
+folder = r"/Users/ismailassafi/Documents/GitHub/paris_1970/assets/images/photos/1538/"
+count = 1
+
+for file_name in os.listdir(folder):
```

Comment:
Take a look at the `tqdm` package, also used by @czheng10 here: https://github.com/dhmit/paris_1970/blob/main/backend/scripts/imageconversion.py#L36
This will allow the user running this script to see progress while iterating